src/constants/indicators.js

```diff
@@ -10,19 +10,19 @@ import {
 // longDescription: shows in the tooltip for a chart
 export default {
   population: {
-    name: 'The population',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Population',
+    shortDescription: 'Total population',
+    longDescription: 'Total population',
     dataType: DATA_TYPES.NUMBER,
     aggregationMethod: AGGREGATION_METHODS.SUM,
     format: '0[.]0a',
     contextual: true,
     source: 'Population, All persons, People',
   },
   growthRate: {
-    name: 'The growthRate',
-    shortDescription: 'TODO',
-    longDescription: 'TODO',
+    name: 'Annual population growth rate',
+    shortDescription: 'Annual growth',
+    longDescription: 'Population growth rate, percentage per annum',
```

Comment:
"per annum" language used here but "per year" used further down. Do they mean the same thing, should they be consistent?

Reply:
Good point.